### PR TITLE
Fix cowswap place order negative balance

### PR DESCRIPTION
**Pull request by yabirgb**
Closes #(issue_number)

## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/docs/blob/main/usage-guides/index.md) to reflect the changes.

### CODE REVIEW

File: rotkehlchen/tasks/historical_balances.py [hunk 1]
```diff
@@ -78,6 +79,13 @@
     (HistoryEventType.TRANSFER, HistoryEventSubType.DONATE),
 }
 
+# CowSwap native asset order deposits and refunds are an escrow lifecycle. The eventual trade
+# tracks the actual spend, so neither side of the escrow transfer should affect wallet balances.
+COWSWAP_ORDER_EVENTS_TO_IGNORE: Final[EventTypeSubtypePairs] = {
+    (HistoryEventType.DEPOSIT, HistoryEventSubType.PLACE_ORDER),
+    (HistoryEventType.WITHDRAWAL, HistoryEventSubType.REFUND),
```
Comment (LefterisJP) [inline rotkehlchen/tasks/historical_balances.py:88]: You are missing the `HistoryEventType.WITHDRAWAL` /  `HistoryEventSubType.CANCEL_ORDER` case. I guess this too shoudl be here. It's the edge case you cancel the order. While the REFUND is when it expires.

Also should add a test for this case.

File: rotkehlchen/tasks/historical_balances.py [hunk 2]
```diff
@@ -78,6 +79,13 @@
     (HistoryEventType.TRANSFER, HistoryEventSubType.DONATE),
 }
 
+# CowSwap native asset order deposits and refunds are an escrow lifecycle. The eventual trade
+# tracks the actual spend, so neither side of the escrow transfer should affect wallet balances.
```
Comment (LefterisJP) [inline rotkehlchen/tasks/historical_balances.py]: the docstring should also mention the cancel order case
